src/App.js

```diff
@@ -1,53 +1,145 @@
 import { useEffect, useState } from 'react';
 import './App.css';
 import ProfileCard from './components/profile-card';
-import profileData from './mock-data';
-import {message} from 'antd'
+import {message, notification, Spin, Tag, Descriptions, Row} from 'antd'
 import _ from 'lodash'
+import { getProfilesData } from './network' 
+
+function mapProfileData(rawData){
+  return rawData.map(({name, email, dob, picture}) => ({
+    name: name.first + ' ' + name.last,
+    email,
+    age: dob.age,
+    imgUrl: picture.large
+  }))
+}
+let remainingCardsNum;
+const REMAINING_CARD_THRESHOLD = 2;
 
 function App() {
   const [profiles, setProfiles] = useState([]);
   const [currentProfileIndex, setProfileIndex] = useState(0);
+  const [viewedProfiles, setViewedProfiles] = useState([]);
```

Comment:
one potential area to improve is not to have viewedProfiles disappear upon reload :)

Reply:
There are lot of improvements to be made. Do you think we should continue to do this in this PR or make another PR for improment? 🤔 If we save it later, I will create new task for this and keep it in backlog